hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -154,50 +158,12 @@ instance FromCBOR CardanoTx where
 instance Arbitrary CardanoTx where
   arbitrary = genUtxo >>= genCardanoTx
 
-genKeyPair :: Gen (Cardano.KeyPair 'Cardano.Payment StandardCrypto)
```

Comment:
Sorry about the moving, but the ordering in this module was triggering me :confused:

Reply:
To be fair, it's a bit of a mess now...